CHANGE-NOTES.md

```diff
@@ -3,6 +3,9 @@
 ## Unreleased
 - Removed automatic rediscovery in case of empty machete file
```

Comment:
Move this to v3.1.1 below

Reply:
Aaand lemme open a task to get rid of this section ✂️

Reply:
#1274

Reply:
ok

Reply:
_słuchaj... okej_